src/storage/BaseProcessor.inl

```diff
@@ -47,6 +57,29 @@ cpp2::ErrorCode BaseProcessor<RESP>::to(kvstore::ResultCode code) {
     }
 }
 
+template <typename RESP>
+cpp2::ErrorCode BaseProcessor<RESP>::writeResultTo(WriteResult code, bool isEdge) {
+    switch (code) {
+    case WriteResult::UNKNOWN_FIELD:
+        if (isEdge) {
+            return cpp2::ErrorCode::E_EDGE_PROP_NOT_FOUND;
+        }
+        return cpp2::ErrorCode::E_TAG_PROP_NOT_FOUND;
+    case WriteResult::NOT_NULLABLE:
+        return cpp2::ErrorCode::E_NOT_NULLABLE;
+    case WriteResult::TYPE_MISMATCH:
+        return cpp2::ErrorCode::E_DATA_TYPE_MISMATCH;
+    case WriteResult::FIELD_UNSET:
+        return cpp2::ErrorCode::E_FIELD_UNSET;
+    case WriteResult::OUT_OF_RANGE:
+        return cpp2::ErrorCode::E_OUT_OF_RANGE;
+    case WriteResult::INCORRECT_VALUE:
+        return cpp2::ErrorCode::E_INVALID_FIELD_VALUE;
+    default:
+        return cpp2::ErrorCode::SUCCEEDED;
```

Comment:
I don't think it's appropriate to make the default branch _SUCCEEDED_

Reply:
Agree, my mistake，thx